Sources/PointFree/Episode/Show.swift

```diff
@@ -794,15 +832,21 @@ private enum EpisodePermission: Equatable {
   }
 }
 
-private let pfVideoClass = CssSelector.class("pf-video")
-private let videoExtraStyles: Stylesheet =
-  pfVideoClass % (
-    width(.pct(100))
-      <> padding(bottom: .pct(56.25))
-      <> position(.relative)
-    )
-    <> (
-      (pfVideoClass > video) % (
-        position(.absolute) <> height(.pct(100))
-      )
-)
+let outerVideoContainerClass: CssSelector =
```

Comment:
we just have to apply these classes to the video elements, but the outer and inner containers.